dashboard/app/controllers/lessons_controller.rb

```diff
@@ -30,6 +30,13 @@ def show
     @lesson_data = @lesson.summarize_for_lesson_show(@current_user, can_view_teacher_markdown?)
   end
 
+  # GET /lessons/2345
+  def show_by_id
```

Comment:
How is this different than edit controller below on line 67?

Reply:
this is showing instead of editing

Reply:
Oh duh 😆